_posts/pandera-dataframe-validation.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: single
+title: "Statistical Data Validation with Pandera"
+excerpt: "A flexible and expressive toolkit for performing statistical validation checks on pandas data structures."
+author: "Niels Bantilan"
+permalink: /posts/pandera-dataframe-validation
+header:
+    overlay_image: images/pandas.jpg
+    overlay_filter: 0.6
+categories:
+  - blog-post
+---
+
+<span style='font-size: 16px;'>
+    Photo credit:
+    <a href="https://www.flickr.com/photos/nationalzoo/5371290900/in/photostream/"
+       target="_blank">
+    Ann Batdorf, Smithsonian's National Zoo</a>
+</span>
+
+Modern data engineering and analysis workflows will often involve using data
+manipulation libraries, which, in the Python universe, would be tools like
+[pandas](https://pandas.pydata.org/) and
+[pyspark](https://spark.apache.org/docs/latest/api/python/index.html). The
+dataframe is a data structure and set of abstractions that enables us to
+express potentially complex transformations on data.
+
+Let's say you're building out an ETL pipeline that cleans raw data into a form
+that can be analyzed or modeled. One problem that you might encounter when
+using these tools is that after multiple transformations, it becomes difficult
+to reason about the contents and properties of a dataframe at any given part of
+the pipeline. This makes the code harder to read, debug and refactor.
+
+In the context of mission-critical analyses or models, it’s vital to ensure the
+quality of the datasets that you're using as inputs to producing key insights
+or predictions.
+
+
+## Pandera: a Statistical Data Validation Toolkit
```

Comment:
Maybe `a Statistical Data Validation Toolkit for Pandas dataframes`
again just silly SEO type things. search engines look hard at headers.